wallet/file-format.md

```diff
@@ -78,7 +79,7 @@ The following table summarize the format:
 | Field | Presence | Format | Comment | Example |
 | ----- | -------- | ------ | ------- | ------- |
 | Version | Mandatory | Integer | Entire part of this specification version | 0 |
-| Nickname | Optional | String || "Savings" |
+| Nickname | Mandatory | String || "Savings" |
```

Comment:
@massalabs/core-team can you check that you can also follwo this constraint ?

Reply:
I'm currently working on using this standard on the node for now we still have our custom standard

Reply:
> I'm currently working on using this standard on the node for now we still have our custom standard

Yes, I know. But do you think that the fact that the nickname is mandatory is blocking on your side ?

Reply:
No I'll just add a default one for now